applications/luci-app-bcp38/root/usr/share/rpcd/acl.d/luci-app-bcp38.json

```diff
@@ -2,10 +2,24 @@
 	"luci-app-bcp38": {
 		"description": "Grant UCI access for luci-app-bcp38",
 		"read": {
-			"uci": [ "bcp38" ]
+			"ubus": {
+				"uci": [
+					"get"
+				]
+			},
+			"uci": [
+				"bcp38"
+			]
 		},
 		"write": {
-			"uci": [ "bcp38" ]
+			"ubus": {
```

Comment:
I think this is not necessary. We already have a write to `uci`  see downside..

```
"uci": [
    "bcp38"
]
```

The same applies to read see above

Reply:
I fix it. I copied that from the example app. Test showed you're right, it works without this. I believed that the ubus section is for authorisation on accessing methods while the latter would be for uci configuration (section).

Reply:
Sorry, I  think I've expressed myself incorrectly.
I meant you should delete the `ubus` but keep the `uci` acl section.
Please have a look to the other apps json acl file for example luci-app-acl or luci-app-acme